web/Services/ErrorService.cs

```diff
@@ -0,0 +1,51 @@
+﻿namespace BricksAndHearts.Services;
+
+public interface IErrorService
+{
+    public (string statusName, string statusMessage) GetStatusMessage(int statusCode);
+}
+public class ErrorService : IErrorService
+{ 
+    public (string statusName, string statusMessage) GetStatusMessage(int statusCode)

```

Comment:
I think these should be in pascal case, i.e. `StatusName` not `statusName`, pedantic I know 😄